src/hardware/RX/Jumper AION Nano RX.json

```diff
@@ -0,0 +1,22 @@
+{
+    "serial_rx": 3,
+    "serial_tx": 1,
+    "radio_busy": 5,
+    "radio_dio1": 4,
+    "radio_miso": 12,
+    "radio_mosi": 13,
+    "radio_nss": 15,
+    "radio_rst": 2,
+    "radio_sck": 14,
+    "radio_dcdc": 1,
+    "power_rxen": 9,
+    "power_txen": 10,
+    "power_min": 0,
+    "power_high": 3,
+    "power_max": 3,
+    "power_default": 3,
+    "power_control": 0,
+    "power_values": [-10,-6,-3,1],
+    "button": 0,
+    "led": 16
+}
```

Comment:
This file is not required, it's is exactly the same as the BETAFPV nano file.

Reply:
We've done some refactoring and now the base target file should be `Generic 2400 PA.json`